_config.yml

```diff
@@ -0,0 +1,52 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: Kroxylicious
+description: Network proxy framework for Apache Kafka
```

Comment:
Not something that needs fixed now, but we should pick a short description for the project. We have one here and a different one in the [pom](https://github.com/kroxylicious/kroxylicious/blob/40d8ed87f76b82188ef08639fdd71d2f3f7bd0a6/pom.xml#L61) neither of which seem like a great answer for our current state.

Reply:
Agreed. Everything I've put in here has come from the existing website in some form or another, but it's definitely worth us revisiting this content.

I foresee many content update PRs in our future 😆